includes/class-sensei-analysis-overview-list-table.php

```diff
@@ -364,13 +367,37 @@ protected function get_row_data( $item ) {
 					$course_title = '<strong><a class="row-title" href="' . esc_url( $url ) . '">' . apply_filters( 'the_title', $item->post_title, $item->ID ) . '</a></strong>';
 				}
 
+				// Fetch learners in course.
+				$course_args           = array(
+					'post_id' => $item->ID,
+					'type'    => 'sensei_course_status',
+					'status'  => 'any',
+				);
+				$course_students_count = Sensei_Utils::sensei_check_for_activity( apply_filters( 'sensei_analysis_course_learners', $course_args, $item ) );
+
+				// Fetch course stats completed lessons and total lessons.
+				$stats                   = Sensei()->course->get_progress_stats( $item->ID );
+				$lesson_completions      = $stats['completed_lessons_count'];
+				$total_lessons           = $stats['lessons_count'];
+				$average_course_progress = 0;
```

Comment:
I've noticed that the default value is a bit inconsistent. When the user is enrolled but hasn't completed any lessons I see `0%`, but `0` otherwise. 

<img width="1293" alt="Screenshot on 2022-03-16 at 23-23-11" src="https://user-images.githubusercontent.com/1612178/158693681-5bfbe324-c13e-4860-89e1-5ab0f65ed3fd.png">

Maybe the default should be also 0%?

Reply:
Ideally, it would be N/A if there is no progress in a given course.

Reply:
Yeah, fixed issue [here](https://github.com/Automattic/sensei/pull/4917/commits/abe5a61ed82d0605a63fdb876bcfd616350764ab#diff-6ce1dc075c4832d5bf050e621bcfecbab91d97504e8bc565f4f476b1ea2f654dR418). 

Moved sprint_f down so it's done when there is no progress

Reply:
I see that you've implemented my suggestion, but I think Donna's makes more sense. It would be better to show N/A when there is no progress.

Reply:
using [NA](https://github.com/Automattic/sensei/pull/4917/commits/f325b3d1c9d4727b344e4eefebe894282ef10d51)